README.md

```diff
@@ -38,6 +38,15 @@ The pipeline is built using [Nextflow](https://www.nextflow.io), a workflow tool
 
 See [usage docs](https://nf-co.re/bcellmagic/usage) for all of the available options when running the pipeline.
 
+## Pipeline Summary
```

Comment:
This probably needs an update :)

Reply:
yes right we moved it from Output to Readme